shopinvader/models/shopinvader_product.py

```diff
@@ -33,12 +33,27 @@ class ShopinvaderProduct(models.Model):
     url_key = fields.Char(
         compute_sudo=True,
     )
+    use_product_shopinvader_name = fields.Boolean(
+        related='backend_id.use_product_shopinvader_name',
+        store=True, readonly=True)
+    shopinvader_name = fields.Char(
+        string='Name',
+        help="Name for shopinvader, if not set the product name will be used.")
+    model_name = fields.Char(compute='_compute_name', readonly=True)
```

Comment:
@bguillot This name `model_name` is really confusing. In my mind, model always refers to a model into odoo...
why not `display_name` or `label` or ???